cookbooks/bcpc-hadoop/attributes/hbase.rb

```diff
@@ -33,7 +33,11 @@
 default['bcpc']['hadoop']['hbase']['ipc']['client']['specificthreadforwriting'] = true
 default['bcpc']['hadoop']['hbase']['client']['primarycalltimeout']['get'] = 100_000
 default['bcpc']['hadoop']['hbase']['client']['primarycalltimeout']['multiget'] = 100_000
+default['bcpc']['hadoop']['hbase']['client']['primarycalltimeout']['scan'] = 1_000_000
```

Comment:
It is suggested to cover it when configuring the client side.
https://hbase.apache.org/book.html#_client_side_properties